arangod/Transaction/Methods.h

```diff
@@ -413,7 +413,7 @@ class Methods {
 
   Result triggerIntermediateCommit();
 
- private:
+  // private:
```

Comment:
Is that intentional?

Reply:
Ah, this was necessary as a temporary workaround to access some private functions. Will fix.